webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -1130,6 +1130,191 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            export_storage: {
+                url: 'environment/export_storage',
+                title: defineMessage({id: 'admin.sidebar.exportStorage', defaultMessage: 'Export Storage'}),
+                isHidden: it.any(
+                    it.not(it.licensedForFeature('Cloud')),
```

Comment:
Is there value in exposing the settings page for on-prem customers?

Reply:
Maybe some day - for now we're leaving it as just for Cloud.

This is really to make getting your data out of Cloud easier - a self hosted customer will have access to their regular S3 bucket as is so it's not super advantageous for them